website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -31,65 +37,124 @@ you'll need to make sure the following outbound traffic is allowed:
 | N/A              | Varies               | `3478`          | STUN            | STUN protocol signaling    |
 | N/A              | Varies               | `49152 - 65535` | TURN            | TURN protocol channel data |
 
-## Resource considerations
+### Where to deploy Gateways
+
+Ideally, Gateways should be deployed as close to the Resources they're serving
+-- in some cases, even on the same physical host. This ensures the lowest
+possible latency and highest possible throughput for client connections, and
+allows you to more easily deploy additional Gateways for a particular Resource
+that needs to handle more client connections.
```

Comment:
```suggestion
possible latency and highest possible throughput for Client connections, and
allows you to more easily deploy additional Gateways for a particular Resource
that needs to handle more Client connections.
```

Can probably capitalize here since The Firezone Client is the only thing that will connect to The Firezone Gateway

Reply:
Well the diff would be huge 🤷‍♀️ either way is fine